server/scripts/setupChainEventListeners.ts

```diff
@@ -181,6 +188,47 @@ const setupChainEventListeners = async (
     });
     return [ node.chain, subscriber ];
   }));
+
+  // Add Erc20 subscribers
+  if (chains === 'all' || chains.includes('erc20')) {
+    const erc20Nodes = await models.ChainNode.findAll({
+      where: {
+        address: {
+          [sequelize.Op.not]: null,
```

Comment:
This wont work, because we use the address field for Moloch/Marlin as well. This query needs to use `include` to ensure that the linked Chain model is a token.